### PR TITLE
Refactor strings in i18n file for upgrade wizard

### PR DESCRIPTION
**Pull request by vwallfahrer**

### CODE REVIEW

**Comment by rsalevsky** [comment]
+1

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 1]
```diff
@@ -558,47 +558,60 @@ en:
...
         blockui_message: 'Aborting upgrade...'
       confirm:
-        header: 'Confirm Download of Upgrade Data'
+        header: 'Reinstall SUSE OpenStack Cloud 6'
         description: |
-          Please reinstall Crowbar with SUSE OpenStack Cloud 6,
-          making sure to use the same configuration used to
-          install SUSE OpenStack Cloud 5.
-        question: 'I have downloaded the upgrade data and I am ready to proceed with the upgrade.'
+          As the next step in the upgrade process you need to redeploy this server
```
Comment (jsuchome) [inline crowbar_framework/config/locales/crowbar/en.yml]: Isn't "this server" bit vague here? Shouldn't we rather talk about "Administration Server" as well?

**Comment by jsuchome** [comment]
+1

File: crowbar_framework/config/locales/crowbar/en.yml [hunk 2]
```diff
@@ -558,47 +558,61 @@ en:
...
+        header_nodes: 'Prepare nodes for upgrade'
+        description_nodes: |
+          As a first step all client nodes of this setup need to be prepared to allow the
+          administration server to be upgraded. Click 'Prepare Nodes for Upgrade' to start
+          that preparation. It is still possible to revert the Upgrade Process after that.
       download:
         cancel: 'Back to selection'
         continue: 'Continue Upgrade Process'
-        abort: 'Abort Upgrade'
+        abort: 'Abort Upgrade and restore client nodes'
```
Comment (aplanas) [inline crowbar_framework/config/locales/crowbar/en.yml]: I am not sure if this a title or not

**Comment by aplanas** [comment]
After normalizing the commented sentence: +1

File: crowbar_framework/config/locales/crowbar/en.yml (hunk 2 above)
Comment (vwallfahrer) [inline crowbar_framework/config/locales/crowbar/en.yml]: @aplanas It is a button to abort the upgrade process.

Comment (aplanas) [inline crowbar_framework/config/locales/crowbar/en.yml]: so all uppercase then? the thing is that we are mixing both in this sentence

Comment (vwallfahrer) [inline crowbar_framework/config/locales/crowbar/en.yml]: You are right, I will change it.

**Pull request merged**